ape_starknet/accounts/__init__.py

```diff
@@ -150,6 +187,12 @@ def genesis_test_accounts(self) -> List:
 
     @property
     def accounts(self) -> Iterator[AccountAPI]:
+        """
+        Iterate over available accounts.
+
+        Returns:
+            Iterator[:class:`~ape.api.AccountAPI`]
```

Comment:
Currently, we can't link back to core. That is something to figure out later.

```suggestion
            Iterator[``AccountAPI``]
```